bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/service/BusinessPartnerMappings.kt

```diff
@@ -60,16 +63,16 @@ class BusinessPartnerMappings {
             externalId = entity.externalId,
             nameParts = entity.nameParts,
             shortName = entity.shortName,
-            legalForm = entity.legalForm,
             identifiers = entity.identifiers.map(::toIdentifierDto),
+            legalForm = entity.legalForm,
             states = entity.states.map(::toStateDto),
             classifications = entity.classifications.map(::toClassificationDto),
             roles = entity.roles,
             postalAddress = toPostalAddressOutputDto(entity.postalAddress),
-            isOwner = entity.isOwner ?: false,      // TODO should be mandatory in entity
-            bpnl = entity.bpnL ?: throw NullPointerException("bpnL is null"),
-            bpns = entity.bpnS,
-            bpna = entity.bpnA ?: throw NullPointerException("bpnA is null"),
+            isOwner = entity.isOwner,
+            bpnL = entity.bpnL ?: throw NullPointerException("bpnL is null"),
```

Comment:
Shouldn't we throw a Bpdm...Exception here?

Reply:
You mean like that:
`throw BpdmNullMappingException(BusinessPartner::class, BusinessPartnerOutputDto::class, BusinessPartnerOutputDto::bpnL, entity.externalId)`